mysite/blog/views.py

```diff
@@ -59,11 +61,16 @@ def Profileview(request,name):
 
 class PostDetailView(DetailView):
     model = Post
+    def get_object(self):
+        obj = super().get_object()
+        obj.view_count += 1
+        obj.save()
+        return obj
 
 
 class PostUpdateView(LoginRequiredMixin, UserPassesTestMixin, UpdateView):
     model = Post
-    fields = ['title', 'content']
+    fields = ['title', 'image', 'content']
```

Comment:
This is unnecessary change for this PR. can you please remove it?

Reply:
This is required because what if any user wants to replace the previously uploaded image with the blog, If I remove this than any user from the front end will not be able to change the uploaded image associated with his blog-post. Though If this feature is not required I will remove it, but then users will only be able to upload the image once.
So, should I remove it. @Kajol-Kumari

Reply:
I have updated the description of this PR, for this particular change.

Reply:
Thanks for letting me know.Let's keep these changes :)